ui/tests/acceptance/client-detail-test.js

```diff
@@ -16,67 +16,84 @@ moduleForAcceptance('Acceptance | client detail', {
     server.create('agent');
     server.create('job', { createAllocations: false });
     server.createList('allocation', 3, { nodeId: node.id });
-
-    visit(`/nodes/${node.id}`);
   },
 });
 
 test('/nodes/:id should have a breadrcumb trail linking back to nodes', function(assert) {
-  assert.equal(findAll('.breadcrumb')[0].textContent, 'Nodes', 'First breadcrumb says nodes');
-  assert.equal(
-    findAll('.breadcrumb')[1].textContent,
-    node.id.split('-')[0],
-    'Second breadcrumb says the node short id'
-  );
-
-  click(findAll('.breadcrumb')[0]);
+  visit(`/nodes/${node.id}`);
+
+  andThen(() => {
+    assert.equal(findAll('.breadcrumb')[0].textContent, 'Nodes', 'First breadcrumb says nodes');
+    assert.equal(
+      findAll('.breadcrumb')[1].textContent,
+      node.id.split('-')[0],
+      'Second breadcrumb says the node short id'
+    );
+  });
+
+  andThen(() => {
+    click(findAll('.breadcrumb')[0]);
+  });
+
   andThen(() => {
     assert.equal(currentURL(), '/nodes', 'First breadcrumb links back to nodes');
   });
 });
 
 test('/nodes/:id should list immediate details for the node in the title', function(assert) {
-  assert.ok(find('.title').textContent.includes(node.name), 'Title includes name');
-  assert.ok(find('.title').textContent.includes(node.id), 'Title includes id');
-  assert.ok(
-    findAll(`.title .node-status-light.${node.status}`).length,
-    'Title includes status light'
-  );
+  visit(`/nodes/${node.id}`);
+
+  andThen(() => {
+    assert.ok(find('.title').textContent.includes(node.name), 'Title includes name');
```

Comment:
oh nice I need to do this instead of `equal(find().text().trim()), 'some text')`

Reply:
IMO `equal` is better than `ok` if you can use it without too much hassle since it shows a diff when the assertion fails.